include/deal.II/lac/precondition.h

```diff
@@ -399,6 +536,16 @@ class PreconditionUseMatrix : public Subscriptor
  * Base class for other preconditioners. Here, only some common features
  * Jacobi, SOR and SSOR preconditioners are implemented. For preconditioning,
  * refer to derived classes.
+ *
+ * The relaxation parameter can be set manually or can be automatically
+ * determined by:
+ * @f[
+ *    \alpha^n_1 := \frac{2}{\lambda_{\max}+\lambda_{\min}}.
+ * @f]
+ * For this purpose, the user needs to set the relaxation parameter to zero.
+ * Internally, the minimum and maximum eigenvalues of the preconditioned
+ * system are determined. For more details on the unterlying algorithms,
+ * see PreconditionChebyshev.
```

Comment:
Since we use the following text in `PreconditionChebyshev`, https://github.com/dealii/dealii/blob/26e124a2a58bfc3fb1ce784884b906fcbd736b44/include/deal.II/lac/precondition.h#L1804-L1808 I think it might make sense to explicitly mention the factor 1.2 in the eigenvalue estimate here as well, e.g.
```suggestion
 * For this purpose, the user needs to set the relaxation parameter to zero.
 * Internally, the minimum and maximum eigenvalues of the preconditioned
 * system are estimated by an eigenvalue algorithm, and the resulting estimate
 * is multiplied by the 1.2 for safety reasons. For more details on the unterlying algorithms,
 * see PreconditionChebyshev.
```